### DISCUSSION TITLE
enable `MultiSelectionPrompt` and `SelectionPrompt` to show Renderables as select items.

### DISCUSSION THREAD

**Discussion by nils-a** [spectreconsole/spectre.console#556] [Ideas]
Having thought a bit about https://github.com/spectreconsole/spectre.console/discussions/550 - is there a particular reason for having the  `Converter` as `Action<T, string>` - I feel `Action<T, IRenderable>` should also be possible. 

(Not sure, why I would add a progress bar as an item in a prompt, but I see no obvious reason why it shouldn't work)

If you don't mind, I'd think I'd like to take a stab at it. 😁 
(That change would probably be breaking, though.)

**Comment by patriksvensson** [comment]
@nils-a go ahead and take a stab at it 😄

**Comment by Armando-CodeCafe** [comment]
Was this ever implemented? would be dope as i have an array of panels i would love to scroll through using this without having to instead just "serialize" them myself first and just using a regular selectionprompt
